From 0909bebf770e9bc4f73431374beafba8103cd26c Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 21 Oct 2021 09:21:26 +0530
Subject: [PATCH] Allow Apps from untrusted sources to install

This patch will allow apps from untrusted source
to install and will not ask to enable the settings
for the same.
This is the customer requirement.

Tracked-On: OAM-99340
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 .../com/android/packageinstaller/PackageInstallerActivity.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/PackageInstaller/src/com/android/packageinstaller/PackageInstallerActivity.java b/packages/PackageInstaller/src/com/android/packageinstaller/PackageInstallerActivity.java
index 665d262d8067..94846a5b36de 100644
--- a/packages/PackageInstaller/src/com/android/packageinstaller/PackageInstallerActivity.java
+++ b/packages/PackageInstaller/src/com/android/packageinstaller/PackageInstallerActivity.java
@@ -429,7 +429,7 @@ public class PackageInstallerActivity extends AlertActivity {
             return;
         }
 
-        if (mAllowUnknownSources || !isInstallRequestFromUnknownSource(getIntent())) {
+        if (true || mAllowUnknownSources || !isInstallRequestFromUnknownSource(getIntent())) {
             initiateInstall();
         } else {
             // Check for unknown sources restrictions.
-- 
2.17.1

